forecasttools/to_flusight.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime, timedelta
+
+import polars as pl
+
+
+# NOTE: needs to change
+def calculate_epidate(date):
+    iso_calendar = date.isocalendar()
+    return iso_calendar[1], iso_calendar[0]
+
+
+def get_flusight_target_end_dates(
+    reference_date: str, horizons=None
+) -> pl.DataFrame:
+    # default horizons
+    if horizons is None:
+        horizons = list(range(-1, 4))
+    reference_date_dt = datetime.strptime(reference_date, "%Y-%m-%d")
+    # list of horizons, target end dates, epiweeks, and epiyears
+    data = []
+    for horizon in horizons:
+        target_end_date = reference_date_dt + timedelta(weeks=horizon)
+        epiweek, epiyear = calculate_epidate(target_end_date)
+        data.append(
+            {
+                "reference_date": reference_date,
+                "target": "wk inc flu hosp",
+                "horizon": horizon,
+                "target_end_date": target_end_date.date(),
+                "epiweek": epiweek,
+                "epiyear": epiyear,
+            }
+        )
+    return pl.DataFrame(data)
+
+
+def get_flusight_table(
+    quantile_forecasts: pl.DataFrame,
+    reference_date: str,
+    quantile_value_col: str = "quantile_value",
+    quantile_level_col: str = "quantile_level",
+    location_col: str = "location",
+    epiweek_col: str = "epiweek",
+    epiyear_col: str = "epiyear",
+    horizons=None,
+    excluded_locations=None,
+) -> pl.DataFrame:
+    # default horizons and locations
+    if horizons is None:
+        horizons = list(range(-1, 4))
+    if excluded_locations is None:
+        excluded_locations = ["60", "78"]
```

Comment:
Why these as default?

Reply:
Grandfathered over from `cfa-forecasttools` at least one might be Virgin Islands. @dylanhmorris